BitwardenShared/Core/Vault/Repositories/VaultRepository.swift

```diff
@@ -1394,15 +1400,17 @@ extension DefaultVaultRepository: VaultRepository {
         }
 
         var sections = [VaultListSection]()
-        if #available(iOSApplicationExtension 17.0, *),
-           let fido2Section = try await loadAutofillFido2Section(
-               availableFido2Credentials: availableFido2Credentials,
-               mode: mode,
-               rpID: rpID,
-               searchText: searchText,
-               searchResults: searchText != nil ? ciphers : nil
-           ) {
-            sections.append(fido2Section)
+        if #available(iOSApplicationExtension 17.0, *) {
```

Comment:
TODO: Need to figure out how the fido2 flow would work. @coroiu do we need to fully decrypt items, or could we expose the needed details in `CipherListView`?